### PR TITLE
implement SelectAll

### PR DESCRIPTION
**Pull request by shogo82148**

### CODE REVIEW

**Review by furusax0621 — APPROVED** [review]
ﾖｼｯ

File: README.md [hunk 1]
```diff
@@ -130,6 +130,26 @@ func SelectUser(ctx context.Context, queryer queryer, primaryKeys *User) (*User,
...
+func SelectAllUser(ctx context.Context, queryer queryer) ([]*User, error) {
+	var ret []*User
+	rows, err := queryer.QueryContext(ctx, "SELECT `id`, `name`, `created_at` FROM `user` ORDER BY `id`")
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	for rows.Next() {
+		var v Foo1
+		if err := rows.Scan(&v.ID, &v.Name, &v.CreatedAt); err != nil {
```
Comment (nao1215) [inline README.md:142]: is this typo ?
```suggestion
	for rows.Next() {
		var v User
		if err := rows.Scan(&v.ID, &v.Name, &v.CreatedAt); err != nil {
```

**Review by nao1215 — APPROVED** [review]
LGTM
You decide if you want to modify typo or not.

File: README.md (hunk 1 above)
Comment (shogo82148) [reply README.md:142]: :hayathinking:

**Pull request merged**